runtime/README.md

```diff
@@ -533,8 +532,8 @@ Command Code: `0x494E_464F` ("INFO")
 
 ## Checksum
 
-For every command, the request and response feature a checksum. This mitigates
-glitches between clients and Caliptra.
+For every command, the request (except for FW_LOAD) and response feature a checksum. This
```

Comment:
nit
```suggestion
For every command except for FW_LOAD, the request and response feature a checksum. This
```

Reply:
Admittedly I was under the impression this only applied to the input/request args at first. That is fixed now and this comment matches your suggestion.